aten/src/ATen/core/boxing/KernelFunction.h

```diff
@@ -90,6 +95,12 @@ class TORCH_API KernelFunction final {
       BoxedKernel::BoxedKernelFunction_withDispatchKeys;
 
   KernelFunction();
+  ~KernelFunction();
+
+  KernelFunction(const KernelFunction&) = default;
+  KernelFunction& operator=(const KernelFunction&) = default;
+
+  KernelFunction(KernelFunction&&) noexcept = default;
```

Comment:
testing whether this line fixes 

```
␛[K/var/lib/jenkins/workspace/torch/library.h:840:17:␛[K ␛[Kerror: ␛[Kuse of deleted function ‘␛[Ktorch::CppFunction::CppFunction(torch::CppFunction&&)␛[K’
2025-07-16T20:31:39.5168847Z   840 |     CppFunction ␛[Kf␛[K((std::forward<Func>(raw_f)));
2025-07-16T20:31:39.5169347Z       |                 ␛[K^␛[K
2025-07-16T20:31:39.5171059Z ␛[K/var/lib/jenkins/workspace/torch/library.h:212:3:␛[K ␛[Knote: ␛[K‘␛[Ktorch::CppFunction::CppFunction(torch::CppFunction&&) noexcept␛[K’ is implicitly deleted because its exception-specification does not match the implicit exception-specification 
```